.travis.yml

```diff
@@ -1,58 +1,98 @@
-# make it explicit that we favor the new container-based travis workers
+# Make it explicit that we favor the
+# new container-based Travis workers
 language: python
+dist: xenial
 
 cache:
   apt: true
   directories:
-  - $HOME/.cache/pip
-  - $HOME/.ccache
-
-dist: xenial
+    - $HOME/.cache/pip
+    - $HOME/.ccache
 
 env:
   global:
-    # Directory where tests are run from
-    - TEST_DIR=/tmp/sklearn
+    - CPU_COUNT=3
+    - TEST_DIR=/tmp/sklearn  # Test directory for continuous integration jobs
+    - PYTEST_VERSION=latest
     - OMP_NUM_THREADS=2
     - OPENBLAS_NUM_THREADS=2
-    - PYTEST_VERSION=latest
+    - SKLEARN_BUILD_PARALLEL=3
+    - SKLEARN_SKIP_NETWORK_TESTS=1
+    # Custom environment variables for the wheel builder
+    - CIBW_TEST_REQUIRES="pytest pytest-xdist"
+    - CIBW_TEST_COMMAND="bash {project}/build_tools/travis/test_wheels.sh"
+    - CIBW_ENVIRONMENT="CPU_COUNT=8
+                        OMP_NUM_THREADS=2
+                        OPENBLAS_NUM_THREADS=2
+                        SKLEARN_BUILD_PARALLEL=8
+                        SKLEARN_SKIP_NETWORK_TESTS=1"
+    # Nightly upload token and staging upload token are set in Travis settings
+    - SCIKIT_LEARN_NIGHTLY_UPLOAD_TOKEN=__token__
```

Comment:
In this case can we set `CIBW_BUILD_VERBOSITY=1`, so it does not fail in the future?

Reply:
I think so :)